logicmonitor/schemata/cloud_services_schema.go

```diff
@@ -15,806 +14,807 @@ func CloudServicesSchema() map[string]*schema.Schema {
 			},
 			Optional: true,
 		},
-
+		
```

Comment:
It looks like this file only has whitespace changes, which are not required; please revert the changes for this file